TestAll.ps1

```diff
@@ -0,0 +1,132 @@
+<#  
+.SYNOPSIS
+    Run the TAEF dlls generated by WinAppSDK
+
+.DESCRIPTION
+    The TestAll script will take the folder input and look for subfolders containing a .testdef file. WinAppSDK
```

Comment:
It may not be obvious for new developers that they can get their teaf test to run in the pipeline by onboarding them via the .testdef file. I presume the developer can also manually run TestAll.cmd on a local dev machine to get onboarded taef tests to run?
I recommend either update readme.md at the root of this repo, or add a readme.md under the \test folder, to make this test onboarding process more discoverable.

Reply:
Readme? Too low a bar. This information belongs @ https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines.md

1. Add new page https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/Tests.md (sp?)
2. Add new link to that @ https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines.md

The docs should explain how to author and use/run *.testdef files

Mention tests can be run via VS' Test Explorer or command line then provide another chapter (or link to another doc) about running tests in the pipeline.

Reply:
After this PR, I'm planning on writing a doc to go into Coding Guidelines

Reply:
Sounds good. But how discoverable is Coding-Guidelines.md itself? I suspect most new developers would first look for readme.md. So, it might still be worthy to put a link to Coding-Guidelines.md in readme.md. Again, this is a nonblocking suggestion.

Reply:
> how discoverable is Coding-Guidelines.md itself?

Good question. We have a gap but it's easy to fix. I'll post a PR

Reply:
Added the missing link(s). Plus some assorted fixing and polish while I was in the neighborhood

https://github.com/microsoft/WindowsAppSDK/pull/3409